UKHomeOffice/sgmr-service
# SPCR-534 Create a new draft report and maintain original report on 'change'

### AC / Description of changes

When user changes a submitted report, it should now create a new draft report and leave the original report in submitted state


### To Test

- create a voyage
- submit the voyage
- go back to this voyage
- click change on any field
- make a change
- click save changes
- > you should be returned to the CYA page
- > you should see your changes
- > you should see all 3 buttons at the bottom (submit, save draft, cancel)

Repeat the above for each 'change' section

- go to manage reports
- > you should see the original report in submitted
- > you should see the new report in drafts
- go to your draft report
- submit the report
- go to manage reports
- > you should see both in submitted **SEE NOTES



### Notes

Next PR will cancel the original submitted report when user submits an updated report

---

**Remember to merge to feature branches not `master`.**
